recipes/rtmidi/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class RtMidiConan(ConanFile):
+    name = "rtmidi"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.music.mcgill.ca/~gary/rtmidi/"
+    description = "Realtime MIDI input/output"
+    topics = ("midi")
+    license = "Copyright (c) 2003-2019 Gary P. Scavone"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os == "Linux":
+            self.requires("libalsa/1.2.4")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+            destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+    def package_info(self):
+        self.cpp_info.libs = ["rtmidi"]
```

Comment:
Please set the name for the `pkg_config` generator explicitly, even if it is `rtmidi`.
It gives maintainers the signal that people have thought about it.
```suggestion
        self.cpp_info.names["pkg_config"] = "rtmidi"
        self.cpp_info.libs = ["rtmidi"]
```